services/uploads/src/antivirus.js

```diff
@@ -32,13 +32,26 @@ async function isS3FileTooBig(s3ObjectKey, s3ObjectBucket) {
   return fileSize > constants.MAX_FILE_SIZE;
 }
 
+let downloadDir = constants.DOWNLOAD_DIR;
```

Comment:
```suggestion
const downloadDir = constants.DOWNLOAD_DIR;
```

Do we alter this anywhere, or good to make this a const?